tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -273,3 +344,37 @@ SPECTRE_TEST_CASE("Unit.ApparentHorizons.StrahlkorperGr.RicciScalar",
       },
       [](const size_t size) noexcept { return DataVector(size, 0.5); });
 }
+
+SPECTRE_TEST_CASE("Unit.ApparentHorizons.StrahlkorperGr.AreaElement",
+                  "[ApparentHorizons][Unit]") {
+  // Check value of dA for a Schwarzschild horizon and a sphere in flat space
+  test_area_element(
+      EinsteinSolutions::KerrSchild{4.0, {{0.0, 0.0, 0.0}}, {{0.0, 0.0, 0.0}}},
+      8.0, [](const size_t size) noexcept { return DataVector(size, 64.0); });
+  test_area_element(
+      EinsteinSolutions::Minkowski<3>{},
+      2.0, [](const size_t size) noexcept { return DataVector(size, 4.0); });
+
+  // Check the area of a Kerr horizon
+  constexpr int l_max = 20;
+  const double mass = 4.444;
+  const std::array<double, 3> spin{{0.4, 0.33, 0.22}};
+  const std::array<double, 3> center{{0.0, 0.0, 0.0}};
+
+  const double kerr_horizon_radius =
+      mass * (1.0 + sqrt(1.0 - square(magnitude(spin))));
+  // Eq. (26.84a) of Thorne and Blandford
+  const double expected_area = 8.0 * M_PI * mass * kerr_horizon_radius;
+
+  const auto horizon_radius = TestHelpers::Kerr::horizon_radius(
+      Strahlkorper<Frame::Inertial>(l_max, l_max, 2.0, center)
+          .ylm_spherepack()
+          .theta_phi_points(),
+      mass, spin);
+
+  const auto kerr_horizon =
```

Comment:
This is just an FYI, I'm fine with the code. You could also write:
```cpp
const Strahlkorper<Frame::Inertial> kerr_horizon(l_max, l_max, get(horizon_radius), center);
```
which might look more familiar to you from SpEC code. Both end up doing the same thing.